lib/checkbufferoverrun.cpp

```diff
@@ -925,7 +925,7 @@ void CheckBufferOverrun::objectIndex()
                     });
                     if (idxValues.empty() ||
                     std::any_of(idxValues.begin(), idxValues.end(), [&](const ValueFlow::Value& vidx) {
-                    if (vidx.isImpossible())
+                        if (vidx.isImpossible())
```

Comment:
won't this be changed back by astyle? did you try that.

Reply:
No longer an issue with uncrustify.